From 6cb5fcafb04c515f8773f81d6867efcedd95688e Mon Sep 17 00:00:00 2001
From: Michael Natterer <mitch@gimp.org>
Date: Mon, 03 Feb 2014 22:05:56 +0000
Subject: Bug 721249 - pdb.gimp_context_set_brush_size() dont't work

Don't reset the brush size before each PDB paint stroke.

(cherry picked from commit 9fe869236bbf178ac9699e0e789f78aa085e5ef6)
---
diff --git a/app/pdb/paint-tools-cmds.c b/app/pdb/paint-tools-cmds.c
index d1d6de3..ba1c826 100644
--- app/pdb/paint-tools-cmds.c
+++ app/pdb/paint-tools-cmds.c
@@ -62,10 +62,6 @@ paint_tools_stroke (Gimp              *gimp,
 
   n_strokes /= 2;  /* #doubles -> #points */
 
-  /* FIXME: i'm most certain that this is wrong, see bug 721249 --mitch */
-  gimp_paint_options_set_default_brush_size (options,
-                                             gimp_context_get_brush (context));
-
   /*  undefine the paint-relevant context properties and get them
    *  from the current context
    */
diff --git a/tools/pdbgen/pdb/paint_tools.pdb b/tools/pdbgen/pdb/paint_tools.pdb
index 8544a56..d084e25 100644
--- tools/pdbgen/pdb/paint_tools.pdb
+++ tools/pdbgen/pdb/paint_tools.pdb
@@ -987,10 +987,6 @@ paint_tools_stroke (Gimp              *gimp,
 
   n_strokes /= 2;  /* #doubles -> #points */
 
-  /* FIXME: i'm most certain that this is wrong, see bug 721249 --mitch */
-  gimp_paint_options_set_default_brush_size (options,
-                                             gimp_context_get_brush (context));
-
   /*  undefine the paint-relevant context properties and get them
    *  from the current context
    */
--
cgit v0.9.2
